lib/ap/plan9/malloc.c

```diff
@@ -37,7 +37,7 @@ struct Arena
 };
 static Arena arena;
 
-#define datoff		((int)((Bucket*)0)->data)
+#define datoff		((long)((Bucket*)0)->data)
 #define nil		((void*)0)
```

Comment:
Comes from original. In fact sources under ap/plan9 keep many things from original system. I think this is an example of "we don't need to change every source to adapt it to posix", no problem for me. It can be changed.